app/league/templates/admin/_formhelpers.html

```diff
@@ -0,0 +1,13 @@
+{% macro render_field(field) %}
```

Comment:
This macro is a neat idea, but I'm not too crazy about how it looks, at least with current styling:

![image](https://cloud.githubusercontent.com/assets/3718321/22866499/8fb818a8-f145-11e6-8320-81b779e8a5e8.png)


I know the overhead out-of-the-box warnings aren't anything wonderful, but I'd rather avoid fragmenting our error handling. I'd prefer to drop this for now, and maybe collaborate on a separate issue on a uniform mechanism for form errors.